### PR TITLE
Add contributors guide

### PR DESCRIPTION
**Pull request by monotron**
This PR adds a contributors guide to help new contributors to look at the standards we have in place for contributions to the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Contributing to OpenChart
+
+Thanks for your interest in contributing to OpenChart. This guide aims to outline some of the standards we've set for contributions to the project.
+
+## Coding standards
+In order to keep the standards of our codebase high, we ask that any code you write adheres to the the [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions). This helps to keep everything consistent and readable.
```
Comment (Kangaroux) [inline CONTRIBUTING.md:6]: Probably slim this down to one sentence

"In order to keep the codebase consistent and easy to maintain, we ask that you please adhere to the recommended <C# Coding Conventions>."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Contributing to OpenChart
+
+Thanks for your interest in contributing to OpenChart. This guide aims to outline some of the standards we've set for contributions to the project.
+
+## Coding standards
+In order to keep the standards of our codebase high, we ask that any code you write adheres to the the [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions). This helps to keep everything consistent and readable. 
+
+Also, in the interest of maintaining proper documentation, we require that any public classes, methods or properties include [XMLdocs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/). Please include a description of what the method/class/property is for, as well as any method arguments. You can usually generate these automatically by typing three forward slashes (`///`) - in Visual Studio Code, [there is an extension to facilitate this](https://marketplace.visualstudio.com/items?itemName=k--kato.docomment).
```
Comment (Kangaroux) [inline CONTRIBUTING.md:8]: For these paragraphs I'd put in a sub header that calls out what the paragraph is talking about. Helps break up the blocks of text a bit and makes it easier to reference.

Comment (Kangaroux) [reply CONTRIBUTING.md:8]: Also, minor wording, but "there is an extension to **help** facilitate this" sounds a bit better

Comment (monotron) [reply CONTRIBUTING.md:8]: I have broken up this section into subsections to make it easier to digest. However:

> help facilitate this

is redundant - it's two words that mean exactly the same thing. If you still want me to reword this then I can think of something else.

**Pull request merged**